src/main/kotlin/link/kotlin/repo/Application.kt

```diff
@@ -18,7 +18,7 @@ fun main() {
     val config = RepoRedirectHandler::class.java.classLoader
         .getResourceAsStream("index.yml")!!
         .use { mapper.readValue(it, object : TypeReference<Map<String, List<String>>>() {}) }
-        .flatMap { it.value.map { group -> group to it.key } }
+        .flatMap { it.value.map { group -> group to it.key.dropLastWhile { it == '/' } } }
```

Comment:
How it works compare to removeSuffix("/")?

Reply:
It will remove all last `/`

Reply:
In fact, they should be the same (cause I do not think that you will accept PRs with a lot of `/` at the end of repo), but `dropLastWhile` will help to avoid some mistakes in this aspect